examples/rfcavity/input_rfcavity_ref_part.in

```diff
@@ -0,0 +1,84 @@
+###############################################################################
+# Particle Beam(s)
+###############################################################################
+beam.npart = 10000  # outside tests, use 1e5 or more
+beam.units = static
+beam.kin_energy = 230
+beam.charge = 1.0e-10
+beam.particle = electron
+beam.distribution = waterbag
+beam.lambdaX = 0.352498964601e-3
+beam.lambdaY = 0.207443478142e-3
+beam.lambdaT = 0.70399950746e-4
+beam.lambdaPx = 5.161852770e-6
+beam.lambdaPy = 9.163582894e-6
+beam.lambdaPt = 0.260528852031e-3
+beam.muxpx = 0.5712386101751441
+beam.muypy = -0.514495755427526
+beam.mutpt = -5.05773e-10
+
+
+###############################################################################
+# Beamline: lattice elements and segments
+###############################################################################
+lattice.elements = monitor dr1 dr2 rf dr2 dr2 rf dr2 dr2 rf dr2 dr2 rf dr2 monitor
+
+monitor.type = beam_monitor
+monitor.backend = h5
+
+dr1.type = drift
+dr1.ds = 0.4
+dr1.nslice = 1
+
+dr2.type = drift
+dr2.ds = 0.032997
+dr1.nslice = 1
+
+rf.type = rfcavity
+rf.ds = 1.31879807
+rf.escale = 62.0
+rf.freq = 1.3e9
+rf.phase = 85.5
+rf.mapsteps = 100
+rf.nslice = 4
+rf.cos_coefficients =                    \
+                0.1644024074311037       \
+                -0.1324009958969339      \
+                4.3443060026047219e-002  \
+                8.5602654094946495e-002  \
+                -0.2433578169042885      \
+                0.5297150596779437       \
+                0.7164884680963959       \
+                -5.2579522442877296e-003 \
+                -5.5025369142193678e-002 \
+                4.6845673335028933e-002  \
+                -2.3279346335638568e-002 \
+                4.0800777539657775e-003  \
+                4.1378326533752169e-003  \
+                -2.5040533340490805e-003 \
+                -4.0654981400000964e-003 \
+                9.6630592067498289e-003  \
+                -8.5275895985990214e-003 \
+                -5.8078747006425020e-002 \
+                -2.4044337836660403e-002 \
+                1.0968240064697212e-002  \
+                -3.4461179858301418e-003 \
+                -8.1201564869443749e-004 \
+                2.1438992904959380e-003  \
+                -1.4997753525697276e-003 \
+                1.8685171825676386e-004
+rf.sin_coefficients = 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0  \
+                0 0 0 0 0 0 0
+
+
+###############################################################################
+# Algorithms
+###############################################################################
+algo.space_charge = false
```

Comment:
Default is false:
```suggestion
```

Reply:
Hm.  I though we previously decided to expose this setting to the user, in order to avoid confusion.

Reply:
Ah, ok then :)